homeassistant/components/plugwise/strings.json

```diff
@@ -55,6 +55,11 @@
         "name": "Plugwise notification"
       }
     },
+    "button": {
+      "reboot": {
+        "name": "Reboot"
+      }
+    },
```

Comment:
Can we also remove the translation key? It would name it restart duo to the device class translations which makes it more in line with other integrations

Reply:
Plugwise calls is a reboot button. And I see other Integrations use "reboot" as well.

I guess it would be named "restart"?

Reply:
We introduced this later, @joostlek is right, we can just remove it and let the device class handle the translations (which means everything will be translated immediately as well, as it re-uses an existing key).